src/app/chip_data_model.gni

```diff
@@ -191,6 +191,7 @@ template("chip_data_model") {
       } else if (cluster == "application-launcher-server") {
         sources += [
           "${_app_root}/app-platform/ContentApp.cpp",
+          "${_app_root}/app-platform/ContentAppClientCommandSender.cpp",
```

Comment:
this does not seem to depend on generated code. 

Could we instead have a "dep += " and place these in a propper BUILD.gn? That would allow us to eventually build unit tests for example (which datamodel.gni prevents)

Also you should have headers in these build files, so that gn can track dependency and layering rules.

Reply:
reverted this change